src/common/utility.ts

```diff
@@ -62,19 +62,33 @@ export class Utility {
         connection.query(sql, (err, rows) => {
             if (Array.isArray(rows)) {
                 if (rows.some(((row) => Array.isArray(row)))) {
-                    rows.forEach((row) => {
+                    rows.forEach((row, index) => {
                         if (Array.isArray(row)) {
-                            OutputChannel.appendLine(asciitable(row));
+                            vscode.commands.executeCommand(
+                                "vscode.previewHtml",
+                                Utility.getPreviewUri(JSON.stringify(row)),
+                                vscode.ViewColumn.Two,
+                                "Results " + (index + 1)).then(() => { }, (e) => {
+                                    OutputChannel.appendLine(e);
+                                });
                         } else {
                             OutputChannel.appendLine(JSON.stringify(row));
                         }
                     });
                 } else {
-                    OutputChannel.appendLine(asciitable(rows));
+                    vscode.commands.executeCommand(
+                        "vscode.previewHtml",
+                        Utility.getPreviewUri(JSON.stringify(rows)),
+                        vscode.ViewColumn.Two,
+                        "Results").then(() => { }, (e) => {
+                            OutputChannel.appendLine(e);
+                        });

```

Comment:
L67-73 and L79-85 is similar, could we wrap them into one method?

Reply:
Yes,review my new commit.